test/fixtures/service_fixtures.js

```diff
@@ -313,7 +283,16 @@ module.exports = {
     }
 
     if (opts.merchant_details) {
-      service.merchant_details = buildMerchantDetailsWithDefaults(opts.merchant_details)
+      service.merchant_details = _.pick(opts.merchant_details, [
+        'name',
+        'address_line1',
+        'address_line2',
+        'address_city',
+        'address_postcode',
+        'address_country',
+        'email',
+        'telephone_number'
+      ])
```

Comment:
Now that any merchant_details fields can be empty, just return the fields that are passed in in the opts